platform/tests/EducationBenchmarking.Platform.Tests/Benchmark/WhenFunctionReceivesPostComparatorSetRequest.cs

```diff
@@ -0,0 +1,63 @@
+using EducationBenchmarking.Platform.Api.Benchmark;
+using EducationBenchmarking.Platform.Api.Benchmark.Models;
+using EducationBenchmarking.Platform.Api.Benchmark.Requests;
+using EducationBenchmarking.Platform.Shared;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Xunit;
+
+namespace EducationBenchmarking.Platform.Tests.Benchmark;
+
+public class WhenFunctionReceivesPostComparatorSetRequest : ComparatorSetFunctionsTestBase
+{
+    [Fact]
+    public async Task ShouldReturn200OnValidRequest()
+    {
+        Db
+            .Setup(d => d.CreateSet(It.IsAny<ComparatorSetRequest>()))
+            .ReturnsAsync(new ComparatorSet());
+
+        Validator
+            .Setup(v => v.ValidateAsync(It.IsAny<ComparatorSetRequest>(), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(new ValidationResult());
+
+        var result =
+            await Functions.PostComparatorSet(CreateRequestWithBody(new ComparatorSetRequest())) as JsonContentResult;
+
+        Assert.NotNull(result);
+        Assert.Equal(200, result?.StatusCode);
+    }
+
+    [Fact]
+    public async Task ShouldReturn40OOnInvalidRequest()
```

Comment:
Typo in name. should be ShouldReturn400OnInvalidRequest()

Reply:
Nice spot - push a fix